setup.py

```diff
@@ -34,8 +34,15 @@ def run(self):
     "datasets>=2.7.0",
 ]
 
+CLEANVISION_REQUIRE = [
+    # Mainly for Datalab's data storage class.
+    # Still some type hints that require datasets
+    "cleanvision==0.3.0",
```

Comment:
this should be lower bound right?

Reply:
I've intentionally kept it fixed for now, since log statements are delicately balanced. This way an updated version will be intentional and require testing.